cmd/podman/main.go

```diff
@@ -25,19 +25,24 @@ var (
 	exitCode = 125
 )
 
+var cmdsNotRequiringRootless = []string{
+	"help",
+	"version",
+}
+
+func sliceContains(slice []string, s string) bool {
```

Comment:
This function probably should live in utils.go.

Reply:
moved